src/components/TransactionBlock/PlrDaoStakingTransactionBlock.tsx

```diff
@@ -477,7 +477,7 @@ const PlrDaoStakingTransactionBlock = ({
         });
         return offers;
       } catch (e) {
-        setTransactionBlockFieldValidationError(transactionBlockId, 'offer', 'Cannot fetch offers');
```

Comment:
Please don't remove this. This is correct, set the error as false in case we get offers